tests/robot-tests/tests/libs/no_javascript.py

```diff
@@ -0,0 +1,33 @@
+from robot.libraries.BuiltIn import BuiltIn
+sl = BuiltIn().get_library_instance('SeleniumLibrary')
+import os
+import requests
+from bs4 import BeautifulSoup
+
+def user_checks_page_accordions_work_with_no_javascript(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, "html.parser")
+
+    first_accordion = soup.select_one("[data-testid='accordionSection']")
+    if not first_accordion:
+        raise AssertionError('No accordionSection found!')
+
+    first_heading = first_accordion.select_one("#publications-1-heading")
+    if not first_heading:
+        raise AssertionError('accordionSection first heading wasn\'t found!')
+    if not 'aria-expanded' in first_heading.attrs:
+        raise AssertionError('First accordion doesn\'t contain "aria-expanded" attribute')
+
+def user_checks_page_details_dropdowns_work_with_no_javascript(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, "html.parser")
+
+    first_details = soup.select_one("details")
+    if not first_details:
+        raise AssertionError('No details tag found!')
+
+    first_summary = first_details.select_one("summary")
+    if not first_summary:
+        raise AssertionError('First details summary wasn\'t found!')
+    if not 'aria-expanded' in first_summary.attrs:
```

Comment:
We should be asserting that `aria-expanded` is either:
- not in the attributes
- if it is in the attributes, it should be set to true

When `aria-expanded` is false, it implies that the section it relates to should be in the closed state. In non-JS land, everything is automatically opened up rather than closed as there is no way to toggle these states without JS.